components/test_raftstore/src/server.rs

```diff
@@ -335,6 +335,12 @@ impl ServerCluster {
             None
         };
 
+        if cfg.tikv.storage.api_version == 2 {
```

Comment:
Use `cfg.tikv.storage.api_version()`.
Please also be aware of the other PR add a causal observer to `test_raftstore`, see https://github.com/tikv/tikv/pull/12303/files#diff-8b71d304aa79611acfb1c46ecd474e6ae860f8fe140d7ad780cff49a3f47e32dR274.